lib/data/AREA_CODE_META_INFO.yml

```diff
@@ -29,6 +29,7 @@
   :country: CANADA
   :service: y
   :description: ''
+  :latlon: '53.7609,-98.8139'
```

Comment:
do you think it makes sense to have them as separate values?

Reply:
I like the idea of separate values